### ISSUE TITLE
Add Swipe support for hold-to-select design

### ISSUE THREAD

**Issue by adnanhz** [scribe-org/Scribe-iOS#243]
### Terms

- [X] I have searched [open and closed feature requests](https://github.com/scribe-org/Scribe-iOS/issues?q=is%3Aissue+label%3Afeature)
- [X] I agree to follow Scribe-iOS' [Code of Conduct](https://github.com/scribe-org/Scribe-iOS/blob/main/.github/CODE_OF_CONDUCT.md)

### Description

In #26, hold-to-select keys were implemented. I'm not sure if this could or should be implemented independently from a more general swipe typing feature, which could be its own issue...

The current behavior is that if you press and hold the "O" key for example, you're then forced to lift your finger to choose the key you want from the pop-up. My preferred way is to be able to keep my finger on the keyboard and then swipe to the choice I want. 

### Contribution

Let me know if you can relate to this, or if my typing nitpicks are unique to me :). Still no Swift knowledge but might dive in.

**Comment by adnanhz** [comment]
Update: I see this was mentioned in the original issue actually.
> [...] is that the alternate character view persists in Scribe, whereas in system keyboards the user needs to continue to hold and drag to select.

Not sure what is meant by that, is it that the difference options would be part of a menu option?

**Comment by andrewtavis** [comment]
I had #150 for this, @adnanhz! 😝😊 I'll delete that one and we can work from here 😇 Honestly I'd love to have this as it's a major thing that's not what a user would expect. Issue is that I really am not sure how best to implement this, but with that being said there could be a library or a basic method? If you'd like to look into that or how best to implement hold swiping in Swift as some basic research, then that'd be a great help! Might be a good way for you to dip your toes into Swift 😉😉😉

We can keep this one unblocked unlike #150 that was, as although I do like not having a system keyboard glitch out when I'm trying to swipe to ß constantly, we can implement this and just save the version where we hold for later once #16 is done :)
